crates/optimism/node/src/node.rs

```diff
@@ -283,3 +288,19 @@ where
         Ok(handle)
     }
 }
+
+/// A basic optimism consensus builder.
+#[derive(Debug, Default, Clone)]
+#[non_exhaustive]
+pub struct OptimismConsensusBuilder;
+
+impl<Node> ConsensusBuilder<Node> for OptimismConsensusBuilder
+where
+    Node: FullNodeTypes,
+{
+    type Consensus = OptimismBeaconConsensus;
+
+    async fn build_consensus(self, ctx: &BuilderContext<Node>) -> eyre::Result<Self::Consensus> {
+        Ok(OptimismBeaconConsensus::new(ctx.chain_spec()))
+    }
```

Comment:
This is great,

we can now remove all the optimism features from eth beacon consensus in a followup: https://github.com/paradigmxyz/reth/issues/8739

Reply:
Added this fix alongside the other feature cleanup